source/views/stoprint/print-jobs.js

```diff
@@ -92,14 +96,22 @@ class PrintJobsView extends React.PureComponent<Props, State> {
 	}
 
 	renderItem = ({item}: {item: PrintJob}) => (
-		<ListRow onPress={() => this.handleJobPress(item)}>
-			<Title>{item.documentName}</Title>
-			<Detail>
-				{item.usageTimeFormatted} {' • '} {item.usageCostFormatted} {' •  '}
-				{item.totalPages} {item.totalPages === 1 ? 'page' : 'pages'} {'\n'}
-				{item.statusFormatted}
-			</Detail>
-		</ListRow>
+		<Timer
+			interval={60000}
+			moment={true}
+			render={({now}) => (
+				<ListRow onPress={() => this.handleJobPress(item)}>
+					<Title>{item.documentName}</Title>
+					<Detail>
+						Expires: {getTimeRemaining(now, item.usageTimeFormatted)} {' • '}{' '}
```

Comment:
```
Expires {getTimeRemaining(now, item.usageTimeFormatted)}{' • '}
{item.usageCostFormatted}{' • '}
{item.totalPages} {item.totalPages === 1 ? 'page' : 'pages'}
```

- Removes the colon from `Expires:`
- Removes the "Pending Release" line that duplicates the heading
- Removes extraneous spaces from the text